### DISCUSSION TITLE
How to exclude files from being bundled

### DISCUSSION THREAD

**Discussion by Jikol** [parcel-bundler/parcel#6941] [Q&A]
I'm little bit struggling now with replicate my construct from parcel v1, where i used parcel-plugin-html-externals for exclude js files from being bundled and copy script for copy that file to dist folder every time I done a build. But now this functionality does not seem to work, so is there an alternative in version 2 already directly part of it?

What i finally want to make is flag one js file for not being bundled and use it for some configurations variables, when someting who do not understand javascript want to make some changes at build production version of app.

My code from v1:

_project dir_
|- config
¯¯|- config.js
|- css
|- js
¯¯|- index.js
|- index.html
|- package.json

_index.html_
```
...
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <link rel="shortcut icon" href="./media/images/favicon.ico">
    <link rel="stylesheet" href="./css/style.css">
    <script src="config/config.js"></script>
    <script src="./js/index.js"></script>
    <title>Some Title</title>
</head>
...

```
_package.json_
```
{
"devDependencies": {
    "parcel-bundler": "^1.12.5",
    "parcel-plugin-html-externals": "^0.2.0"
  },
  "externals": {
    "config/**/*": false
  }
}
```
And index.html inside `dist ` folder

```
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <link rel="shortcut icon" href="/favicon.02a0819d.ico">
    <link rel="stylesheet" href="/style.78032849.css">
    <script src="config/config.js"></script>
    <title>Some Title</title>
<script src="/style.78032849.js"></script><link rel="stylesheet" href="/js.00a46daa.css"><script src="/js.00a46daa.js"></script></head>
```

**Comment by strtob** [comment]
I have the same topic, a separate config.js which I want to make available in my js code but should be avoided to minify and implement together with other code in one single js file - config.js should be stay alone like the dev phase

**Comment by RichardJECooke** [comment]
Did you find out how to do this please? I need it for Neutralino.
